Makefile

```diff
@@ -21,9 +23,9 @@ test: envtest
 envtest:
 ifndef HAS_SETUP_ENVTEST
 	go install sigs.k8s.io/controller-runtime/tools/setup-envtest@2c3a6fa2996c026b284c7fe2b055274cd9a556bc #v0.13.0
-	@echo "setup-envtest has been installed"
+	$(info "setup-envtest has been installed")
```

Comment:
Why do we want to change `echo` to `info` ?

Reply:
@andreyvelich  I changed them since I think that it might be better to use the Make function instead of shell scripts in Makefile.

Would you like to keep using `@echo`?

Reply:
Sounds good, as I can see [K8s also uses `info`](https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile#L50)